src/viur/core/config.py

```diff
@@ -195,6 +195,9 @@ def __setitem__(self, key, value):
     # Upper limit of the amount of parameters we accept per request. Prevents Hash-Collision-Attacks
     "viur.maxPostParamsCount": 250,
 
+    # Function which decides if a request param should be used or filtered out
+    "viur.paramFilterFunction": lambda key, value: True,
```

Comment:
We should keep the old behavior.
```suggestion
    "viur.paramFilterFunction": lambda key, value: key.startswith("_"),
```